### ISSUE TITLE
fresh 3.0/3.1 install - The SDK 'Microsoft.NET.Sdk' could not be found

### ISSUE THREAD

**Issue by jeske** [xenko3d/xenko#457]
**Release Type**: launcher official release
**Version**: 3.1.0.1-beta02-0550  and 3.0.0.9
**Platform(s)**: Windows
**Describe the bug**

Launcher installed 3.0/3.1 throws errors trying to instantiate any project type. 

**To Reproduce**

1. Installed Xenko Launcher
2. Let it install 3.1 release
3. installed VS plugins (I have both VS 2017/2019 installed)
4. Ran VS installer separately to update both to latest
5. launched Xenko, clicked "New Game", then "Select", then "OK" on project options dialog
6. Creating session window shows errors... project is busted and won't open

I also manually installed .NET Core 3.0, .NET Framework 4.6.2 devpack (referenced in the Xenko docs), and .NET Framework 4.8 devpack. It didn't fix anything. 

https://dotnet.microsoft.com/download/visual-studio-sdks

**Expected behavior**
I expected to be able to spawn a new project correctly

**Screenshots**
**Log and callstacks**

See thread..

https://forums.xenko.com/t/a-bunch-of-3-1-3-0-install-issues/1905

**Comment by Kryptos-FR** [comment]
Did you you install all of those in their default path locations? If not, where?

**Comment by jeske** [comment]
I was able to fix this problem...

VS was already installed in the default paths before my Xenko install, but they were not fully updated. I got some error during the Xenko install, and followed some instructions to separately run the VS installer to update them. However, even after this, and reinstalling the VS plugins, Xenko still gave me the above errors. 

I uninstalled and reinstalled Xenko and now it seems to be working properly... 

(though the first time I opened a Xenko 3.1 project, it said "this might take a minute the first time" .... for TEN minutes.... I ended up killing the process, then restarting, and it loaded up right away)
